### PR TITLE
Content updates on the QC40 page

### PR DESCRIPTION
**Pull request by y4izus**
The event's team requested some changes on the content. Waiting for their review. DO NOT MERGE yet, please :)

### CODE REVIEW

File: components/ui/AppDataTable.vue [hunk 1]
```diff
@@ -68,4 +69,9 @@ export default class AppDataTable extends Vue {
   border-top: none;
   border-bottom: 1px solid $border-color;
 }
+
+.bx--data-table li {
+  list-style-type: disc;
+  margin-left: 1.2rem;
```
Comment (eddybrando) [inline components/ui/AppDataTable.vue:75]: For consistent spacing, please try and use the spacing variables.

I'm afraid there isn't one in https://www.carbondesignsystem.com/guidelines/spacing/overview/ that matches exactly `1.2rem`, but I believe `1.5rem` may be a good alternative:

```suggestion
  margin-left: $spacing-06;
```

**Review by eddybrando — APPROVED** [review]
Thanks for the PR! 🙂 

I have just one small request to use a `$spacing` variable.

File: components/ui/AppDataTable.vue (hunk 1 above)
Comment (y4izus) [reply components/ui/AppDataTable.vue:75]: Good catch! thank you!

File: components/ui/AppMosaic.vue [hunk 2]
```diff
@@ -186,12 +186,18 @@ export default class AppMosaic extends Vue {
...
+          list-style-type: disc;
+          margin-left: $spacing-06;
+        }
       }
 
       &-link {
         display: flex;
         margin-top: $spacing-05;
       }
+
```
Comment (eddybrando) [inline components/ui/AppMosaic.vue:200]: 😬 

```suggestion
```

File: pages/events/physics-of-computation.vue [hunk 3]
```diff
@@ -168,113 +182,174 @@ export default class PhysicsOfComputationPage extends QiskitPage {
...
+        <li>Zaira Nazario (IBM)</li>
+      </ul>
+      `
     }
   ]
 
   // Data for the agenda section
-  agendaSectionTitle: string = 'Event schedule'
+  morningAgendaSectionTitle: string = 'Morning Session'
+  afternoonAgendaSectionTitle: string = 'Afternoon Session'
```
Comment (eddybrando) [inline pages/events/physics-of-computation.vue:223]: The types get automatically inferred by their values:

```suggestion
  morningAgendaSectionTitle = 'Morning Session'
  afternoonAgendaSectionTitle = 'Afternoon Session'
```

**Review by eddybrando — APPROVED** [review]
Awesome, LGMT! Thanks @y4izus

**Pull request merged**